src/pages/admin/volunteers/[volId]/index.tsx

```diff
@@ -36,6 +36,16 @@ const VolunteerPage: NextPage<Props> = ({ vol }) => {
         }
     };
 
+    const handleSendVerificationEmail = async () => {
+        try {
+            if (vol._id) {
+                await fetch(`${urls.api.volunteers}/${vol._id}/email`, { method: "PUT" });
+            }
+        } catch (error) {
+            console.log(error);
+        }
+    };
+
```

Comment:
Does the admin know that the email has been sent successfully? I think having some simple text to let them know the email has been sent might be nice to have. The fetch call should return a Response object, and if its status is 200 you can set a flag to show the text

Reply:
Added functionality to change the text to show that it sent! Take a look and see if it looks ok, sorry it took so long!